tests/puppeteer/campaigns/functional/BO/14_advancedParameters/06_team/profiles/05_sortProfiles.js

```diff
@@ -0,0 +1,96 @@
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const EmployeesPage = require('@pages/BO/advancedParameters/team/index');
+const ProfilesPage = require('@pages/BO/advancedParameters/team/profiles/index');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_advancedParams_team_profiles_sortProfiles';
+
+let browser;
+let page;
+let numberOfProfiles = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    employeesPage: new EmployeesPage(page),
+    profilesPage: new ProfilesPage(page),
+  };
+};
+
+// Sort profiles by id, name
+describe('Sort Profiles', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO
+  loginCommon.loginBO();
+
+  it('should go to "Advanced parameters>Team" page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToAdvancedParamsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.advancedParametersLink,
+      this.pageObjects.boBasePage.teamLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.employeesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.employeesPage.pageTitle);
+  });
+
+  it('should go to "Profiles" page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProfilesPage', baseContext);
+    await this.pageObjects.employeesPage.goToProfilesPage();
+    const pageTitle = await this.pageObjects.profilesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.profilesPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of profiles', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+    numberOfProfiles = await this.pageObjects.profilesPage.resetAndGetNumberOfLines();
+    await expect(numberOfProfiles).to.be.above(0);
+  });
+
+  const tests = [
+    {args: {testIdentifier: 'sortByIDDesc', sortBy: 'id_profile', sortDirection: 'desc'}},
+    {args: {testIdentifier: 'sortByIDAsc', sortBy: 'id_profile', sortDirection: 'asc'}},
+    {args: {testIdentifier: 'sortByNameDesc', sortBy: 'name', sortDirection: 'desc'}},
+    {args: {testIdentifier: 'sortByNameAsc', sortBy: 'name', sortDirection: 'asc'}},
```

Comment:
```suggestion
    {args: {testIdentifier: 'sortByIDDesc', sortBy: 'id_profile', sortDirection: 'desc'}},
    {args: {testIdentifier: 'sortByNameDesc', sortBy: 'name', sortDirection: 'desc'}},
    {args: {testIdentifier: 'sortByNameAsc', sortBy: 'name', sortDirection: 'asc'}},
    {args: {testIdentifier: 'sortByIDAsc', sortBy: 'id_profile', sortDirection: 'asc'}},
```
should be back to default sort value at the end

Reply:
Done thx :)